crates/pallet-domains/src/staking.rs

```diff
@@ -801,58 +842,10 @@ pub(crate) fn do_unlock_funds<T: Config>(
             });
         }
 
-        Ok(amount_to_unlock)
+        Ok(amount_to_release)
```

Comment:
Look like we should return the total amount of fund include both release and mint